### ISSUE TITLE
M+ fonts do not rename, and collide with the originals.

### ISSUE THREAD

**Issue by DevotionGeo** [ToxicFrog/Ligaturizer#58]
M+ 1m and M+ 1mn fonts don't rename. When I opened the fonts in FontForge and looked at the Font info, the name have the 'Liga' prefix, but when I open the same font in font viewer it doesn't have the prefix, and while using inside VSCode it activates with the original name (overriding the original) instead of activating with the prefixed name.

![screenshot from 2019-01-08 05-21-58](https://user-images.githubusercontent.com/7348138/50801168-61e9fe80-1305-11e9-8e38-eb50fdd2e301.png)

![screenshot from 2019-01-08 05-22-39](https://user-images.githubusercontent.com/7348138/50801184-76c69200-1305-11e9-8c2a-0420c375ea66.png)

I also added the following to the prefixed_fonts array (but the problem retained).
```
  'fonts/codeface/fonts/mplus1m/*.ttf',
  'fonts/codeface/fonts/mplus1mn/*.ttf',
```

**Comment by oganm** [comment]
The same is true for ubuntu mono at least

**Comment by Avi-D-coder** [comment]
IBM plex as well

**Comment by ToxicFrog** [comment]
Huh. For me, IBM Plex Mono shows up just fine (as Ligalex Mono), but Ubuntu Mono does not. They have the correct `familyname` and `fullname`, and the `Family`, `UniqueID`, and `Fullname` in `sfnt_names` are also correct.

The only suspicious things I see are these entries in `sfnt_names` for Ubuntu Mono:

    ('English (US)', 'Preferred Family', 'Ubuntu Mono')
    ('English (US)', 'Compatible Full', 'Ubuntu Mono')

And in Plex:

    ('English (US)', 'Preferred Family', 'IBM Plex Mono')

I wonder if the `Preferred Family` is overriding the other settings.

**Comment by ToxicFrog** [comment]
Yep, that's the problem. Should have a fix later today, hopefully.

**Comment by ToxicFrog** [comment]
This should hopefully be fixed by 6d3070ae1c6b1a6ec196db486be813d713c7ce9c.
